File: ruby/sell.rb
# Sell trade example
require_relative 'binary_options_tools_uni'

client = BinaryOptionsToolsUni::PocketOption.new("your-session-id")
sleep 5

balance_before = client.balance
puts "Balance before: $#{balance_before}"

deal = client.sell("EURUSD_otc", 60, 1.0)
puts "Trade placed: #{deal}"

sleep 65

balance_after = client.balance
puts "Balance after: $#{balance_after}"
puts "Profit/Loss: $#{balance_after - balance_before}"

File: kotlin/Sell.kt
// Sell trade example
import binary_options_tools_uni.*
import kotlinx.coroutines.*

suspend fun main() {
    val client = PocketOption.new("your-session-id")
    delay(5000)
    
    val balanceBefore = client.balance()
    println("Balance before: $$balanceBefore")
    
    val deal = client.sell("EURUSD_otc", 60u, 1.0)
    println("Trade placed: $deal")
    
    delay(65000)
    
    val balanceAfter = client.balance()
    println("Balance after: $$balanceAfter")
    println("Profit/Loss: $${balanceAfter - balanceBefore}")
}
